content/_data/changelogs/weekly.yml

```
@@ -11355,6 +11355,31 @@
   # pull: 5551 (PR title: Bump matrix-project from 1.18 to 1.19)
   # pull: 5554 (PR title: Bump parent POM from 1.63 to 1.64)
 
+- version: '2.298'
+  date: 2021-06-15
+  changes:
+  - type: rfe
+    category: rfe
+    pull: 5556
+    authors:
+    - jglick
+    message: |-
+      Documenting REST methods to mark an (agent) node temporarily offline and related tasks.
+  - type: rfe
+    category: rfe
+    pull: 5555
+    authors:
+    - Wadeck
+    message: |-
+      Add `X-Frame-Options` header added to AJAX responses.
```

needs to convert backquotes to <code> tag
```suggestion
      Add <code>X-Frame-Options</code> header to AJAX responses.
```